kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/airport/FlightRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.good.patterns.airport;
+
+import com.kodilla.good.patterns.airport.simple.City;
+
+public class FlightRequest {
+
+    private final City name;
```

Comment:
City nie moze miec nazwy name :/ city to city a name to name

Reply:
Usunąłęm klase City